scipy/spatial/qhull.pyx

```diff
@@ -2410,3 +2449,269 @@ class Voronoi(_QhullUser):
+    halfspaces : ndarray of double, shape (nineq, ndim+1)
+                 Input halfspaces.
+    intersections : ndarray of double, shape (ninter, ndim)
+                    Intersections of all halfspaces.
+    dual_points : ndarray of double, shape (nineq, ndim)
+                  Dual points of the input halfspaces.
+    dual_vertices : ndarray of ints, shape (nvertices,)
+                    Indices of points forming the vertices of the dual convex hull.
+                    For 2-D convex hulls, the vertices are in counterclockwise order.
+                    For other dimensions, they are in input order.
+    dual_simplices : ndarray of ints, shape (nfacet, ndim)
+                     Indices of points forming the simplical facets of the dual convex hull.
+    dual_neighbors : ndarray of ints, shape (nfacet, ndim)
+                     Indices of neighbor facets for each facet.
+                     The kth neighbor is opposite to the kth vertex.
+                     -1 denotes no neighbor.
+    dual_equations : ndarray of double, shape (nfacet, ndim+1)
+                     [normal, offset] forming the hyperplane equation of the dual facet
+                     (see `Qhull documentation <http://www.qhull.org/>`__  for more).
+    coplanar : ndarray of int, shape (ncoplanar, 3)
+        Indices of coplanar points and the corresponding indices of
+        the nearest facets and nearest vertex indices.  Coplanar
+        points are input points which were *not* included in the
+        triangulation due to numerical precision issues.
+
+        If option "Qc" is not specified, this list is not computed.
+    dual_area : float
+        Area of the dual convex hull
+    dual_volume : float
+        Volume of the dual convex hull
+
+    Raises
+    ------
+    QhullError
+        Raised when Qhull encounters an error condition, such as
+        geometrical degeneracy when options to resolve are not enabled.
+    ValueError
+        Raised if an incompatible array is given as input.
+
+    Notes
+    -----
+    The intersections are computed using the 
+    `Qhull library <http://www.qhull.org/>`__.
+
+    Examples
+    --------
+
+    TODO: example
+    Halfspace intersection of planes forming some polygon
+
+    >>> from scipy.spatial import HalfspaceIntersection
+    >>> import numpy as np
+    >>> halfspaces = np.array([[-1, 0., 0.],
+    >>>                        [0., -1., 0.],
+    >>>                        [2., 1., -4.],
+    >>>                        [-0.5, 1., -2.]])
+    >>> feasible_point = np.array([0.5, 0.5])
+    >>> hs = HalfspaceIntersection(halfspaces, feasible_point)
+
+    Plot halsfpaces as filled regions and intersection points:
+
+    >>> import matplotlib.pyplot as plt
+    >>> xlim, ylim = (-1, 3), (-2, 6)
+    >>> x = np.linspace(-1, 3, 100)
+    >>> symbols = ['-', '+', 'x', '*']
+    >>> signs = [0, 0, -1, -1]
+    >>> for h, sym, sign in zip(halfspaces, symbols, signs):
+    >>>   if h.item(1) == 0:
```

Comment:
Continuation lines are written with `...` instead of `>>>`.
Also, 4 space indent also here.